api/src/main/java/org/mandulis/mts/attachment/AttachmentService.java

```diff
@@ -1,20 +1,28 @@
-package org.mandulis.mts.attachment;
-
-import org.springframework.beans.factory.annotation.Autowired;
+package org.mandulis.mts.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.mandulis.mts.attachment.Attachment;
+import org.mandulis.mts.attachment.AttachmentRepository;
+import org.mandulis.mts.dto.AttachmentDto;
+import org.mandulis.mts.exception.TicketNotFoundException;
+import org.mandulis.mts.storage.StorageService;
+import org.mandulis.mts.ticket.Ticket;
+import org.mandulis.mts.ticket.TicketRepository;
 import org.springframework.stereotype.Service;
 
+import java.nio.file.Path;
 import java.util.List;
 import java.util.Optional;
 
+@Slf4j
 @Service
+@RequiredArgsConstructor
 public class AttachmentService {
 
     private final AttachmentRepository attachmentRepository;
-
-    @Autowired
-    public AttachmentService(AttachmentRepository attachmentRepository) {
-        this.attachmentRepository = attachmentRepository;
-    }
+    private final StorageService storageService;
+    private final TicketRepository ticketRepository;
 
     public List<Attachment> findAll() {
```

Comment:
This function should return a DTO and not an entity

Reply:
Sure, fixing